### PR TITLE
docs/site/ko/guide/estimator.ipynd translate

### PR DESCRIPTION
**Pull request by qwert232323**
KO : docs/site/ko/guide/estimator.ipynd

translate estimator.ipynd
EN -> KO

EN : https://github.com/tensorflow/docs/blob/master/site/en/guide/estimator.ipynb

### CODE REVIEW

File: site/ko/guide/estimator.ipynb [hunk 1]
```diff
@@ -0,0 +1,471 @@
...
+    "colab_type": "text",
+    "id": "MfBg1C5NB3X0"
+   },
+   "source": [
+    "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://www.tensorflow.org/guide/estimator\"><img src=\"https://www.tensorflow.org/images/tf_logo_32px.png\" />View on TensorFlow.org</a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/docs/blob/master/site/en/guide/estimator.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />Run in Google Colab</a>\n",
```
Comment (Choiuijin1125) [inline site/ko/guide/estimator.ipynb]: You need to change address Eng doc into Ko :) 'en/guide' → 'ko/guide'

And you need to translate 'Run in Google Colab' as '구글 코랩(Colab)에서 실행하기.

File: site/ko/guide/estimator.ipynb [hunk 2]
```diff
@@ -0,0 +1,471 @@
...
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://github.com/tensorflow/docs/blob/master/site/en/guide/estimator.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />View source on GitHub</a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a href=\"https://storage.googleapis.com/tensorflow_docs/docs/site/en/guide/estimator.ipynb\"><img src=\"https://www.tensorflow.org/images/download_logo_32px.png\" />Download notebook</a>\n",
```
Comment (Choiuijin1125) [inline site/ko/guide/estimator.ipynb]: en/guide → ko/guide
Download notebook → '노트북 다운로드 하기'

File: site/ko/guide/estimator.ipynb [hunk 3]
```diff
@@ -0,0 +1,471 @@
...
+   "source": [
+    "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://www.tensorflow.org/guide/estimator\"><img src=\"https://www.tensorflow.org/images/tf_logo_32px.png\" />View on TensorFlow.org</a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/docs/blob/master/site/en/guide/estimator.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />Run in Google Colab</a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://github.com/tensorflow/docs/blob/master/site/en/guide/estimator.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />View source on GitHub</a>\n",
```
Comment (Choiuijin1125) [inline site/ko/guide/estimator.ipynb]: en/guide → ko/guide
'View source on GitHub' as '깃허브(GitHub) 소스 보기'

File: site/ko/guide/estimator.ipynb [hunk 4]
```diff
@@ -0,0 +1,471 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "MfBg1C5NB3X0"
+   },
+   "source": [
+    "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://www.tensorflow.org/guide/estimator\"><img src=\"https://www.tensorflow.org/images/tf_logo_32px.png\" />View on TensorFlow.org</a>\n",
```
Comment (Choiuijin1125) [inline site/ko/guide/estimator.ipynb]: 'View on TensorFlow.org' as 'TensorFlow.org에서 보기' :)

File: site/ko/guide/estimator.ipynb [hunk 5]
```diff
@@ -0,0 +1,471 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "oEinLJt2Uowq"
+   },
+   "source": [
+    "이 문서에서는 고급 텐서플로 API인 `tf.estimator` 를 소개합니다. 추정기는 다음 작업들을 캡슐화 합니다.\n",
```
Comment (Choiuijin1125) [inline site/ko/guide/estimator.ipynb]: You mostly translate 'High-level' as '고수준' so I suggest '고급 텐서플로 API' → '고수준 텐서플로 API'

File: site/ko/guide/estimator.ipynb [hunk 6]
```diff
@@ -0,0 +1,471 @@
...
+    "\n",
+    "### 사전 제작된 추정기 프로그램의 구조\n",
+    "\n",
+    "사전 제작된 추정기에 의존하는 텐서플로 프로그램은 일반적으로 4단계로 구성됩니다.\n",
+    "\n",
+    "#### 1. 하나 이상의 데이터세트 임포트 함수를 작성합니다.\n",
+    "\n",
+    "예를 들어, 훈련 세트를 가져오는 함수와 테스트 세트를 가져오는 다른 함수를 생성했을 때. 각각의 데이터세트 임포트 함수는 다음 두개의 오브젝트를 반환해야 합니다 :\n",
+    "\n",
+    "* a dictionary in which the keys are feature names and the values are Tensors (or SparseTensors) containing the corresponding feature data\n",
```
Comment (Choiuijin1125) [inline site/ko/guide/estimator.ipynb]: I think you missed one line, please translate this line :)

File: site/ko/guide/estimator.ipynb [hunk 7]
```diff
@@ -0,0 +1,471 @@
...
+    "def input_fn(dataset):\n",
+    "    ...  # 데이터 세트를 조작하여 특성 딕셔너리와 레이블을 추출합니다.\n",
+    "    return feature_dict, label\n",
+    "```\n",
+    "\n",
+    "자세한 내용은 [data guide](../../guide/data.md)를 참조하세요.\n",
+    "\n",
+    "#### 2. 특성 열을 정의합니다.\n",
+    "\n",
+    "각 `tf.feature_column`은 특성 이름, 유형 및 입력 사전처리를 식별합니다. 예를 들어, 다음 예제는 정수 혹은 부동 소수점 데이터를 가지는 3개의 특성 열을 생성합니다. 처음의 두 특성 열은 단순히 특성의 이름과 유형만 식별합니다. 세번째 특성 열은 프로그램이 원시 데이터를 조정하기 위해 호출할 람다도 지정합니다 :\n",
```
Comment (Choiuijin1125) [inline site/ko/guide/estimator.ipynb]: pre-processing is mostly translated as '전처리' so I suggest '사전처리' as '전처리' 
And raw data means unprocessed(original) data. so in my opinion, '원본' is more natural word for 'raw' so I suggest '원시' → '원본' :)

File: site/ko/guide/estimator.ipynb [hunk 8]
```diff
@@ -0,0 +1,471 @@
...
+    "```\n",
+    "# 3개의 특성 열을 정의합니다.\n",
+    "population = tf.feature_column.numeric_column('population')\n",
+    "crime_rate = tf.feature_column.numeric_column('crime_rate')\n",
+    "median_education = tf.feature_column.numeric_column(\n",
+    "  'median_education',\n",
+    "  normalizer_fn=lambda x: x - global_education_mean)\n",
+    "```\n",
+    "\n",
+    "자세한 내용은 [Feature columns tutorial](https://www.tensorflow.org/tutorials/keras/feature_columns)을 참조하세요.\n",
```
Comment (Choiuijin1125) [inline site/ko/guide/estimator.ipynb]: Why don't you translate 'Feature columns tutorial'? it's might be better :)

File: site/ko/guide/estimator.ipynb [hunk 9]
```diff
@@ -0,0 +1,471 @@
...
+    "\n",
+    "예를 들어, 다음 코드는 입력 함수의 기본 골격을 보여줍니다.\n",
+    "\n",
+    "```\n",
+    "def input_fn(dataset):\n",
+    "    ...  # 데이터 세트를 조작하여 특성 딕셔너리와 레이블을 추출합니다.\n",
+    "    return feature_dict, label\n",
+    "```\n",
+    "\n",
+    "자세한 내용은 [data guide](../../guide/data.md)를 참조하세요.\n",
```
Comment (Choiuijin1125) [inline site/ko/guide/estimator.ipynb]: Why don't you translate 'data guide'? it's might be better :)

**Comment by qwert232323** [comment]
번역 리뷰해주신대로 수정 완료했습니다!

File: site/ko/guide/estimator.ipynb [hunk 10]
```diff
@@ -0,0 +1,471 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "oIaPjYgnZdn6"
+   },
+   "source": [
+    "## 사용자 지정 추정기\n",
+    "\n",
+    "모든 추정기의 핵심은 훈련, 평가 및 예측을 위한 그래프를 만드는 방법인 *모델 함수* 입니다. 사전 제작된 추정기를 사용하는 경우, 다른 누군가가 이미 모델 함수를 구현했습니다. 사용자 지정 추정기를 사용할 때에는 모델 함수를 직접 작성해야 합니다.\n",
```
Comment (Choiuijin1125) [inline site/ko/guide/estimator.ipynb]: I'm a bit feeling awkward '다른 누군가가 이미 모델 함수를 구현했습니다.'
I think it's better that '다른 누군가가 이미 모델 함수를 구현해놓았습니다.
How about your opinion ? :)

**Comment by qwert232323** [comment]
@googlebot  i signed it!

**Comment by lamberta** [comment]
Hi qwert232323@, it looks like your commit is using a different email address than what you used for the CLA. Can you sign the CLA with the other email, as well?

**Comment by qwert232323** [comment]
> Hi @qwert232323 , it looks like your commit is using a different email address than what you used for the CLA. Can you sign the CLA with the other email, as well?

I've tried various ways but it doesn't seem to work.

**Comment by rickiepark** [comment]
Hi @qwert232323, Can you sign the CLA with @qwert232324 's email?
If it's not possible, please close this PR and recreate it.
Thank you!

**Pull request closed without merging**